index.html

```diff
@@ -56,7 +56,7 @@
         {
           name: "Aaron Gustafson",
           company: "Microsoft Corporation",
-          companyURL: "https://microsoft.com/",
+          companyURL: "https://www.microsoft.com/en-us/",
```

Comment:
Is this the url we should use @aarongustafson ? or is there a better one?

Reply:
There is also another Microsoft contributor that uses the same URL. If there is a better URL to use, should I also update this person's Microsoft URL? Thanks.

https://github.com/w3c/manifest/blob/d2e20c9ab0983ec9d17c2434c19e27ea0f9359b6/index.html#L18-L24

Reply:
OK let's go with that for now then

Reply:
I would suggest the root domain as not everyone who follows it will have en-us as their primary language.